camel-idea-plugin/src/main/java/com/github/cameltooling/idea/runner/debugger/CamelDebuggerPatcher.java

```diff
@@ -340,7 +340,13 @@ private static File generatePomFileWithCamelDebugger(ExecutionMode mode, Project
         final Model model;
         final ParametersList parametersList = parameters.getProgramParametersList();
         final int index = Math.max(parametersList.getParameters().indexOf("-f"), parametersList.getParameters().indexOf("--file"));
-        String targetFileName = index == -1 ? "pom.xml" : parametersList.get(index + 1);
+        String targetFileName;
+        if (index == -1) {
+            targetFileName = "pom.xml";
+        } else {
+            targetFileName = parametersList.get(index + 1);
+        }
```

Comment:
Not sure I understand why the ternary operator is not good here but it is good below. This sounds like a controversial change so could you please revert?